lambda_layers/core_layer/python/core_layer/alembic/versions/1594a357d751_modified_item_model_to_output_tags_.py

```diff
@@ -18,7 +18,7 @@
 
 def upgrade():
     # ### commands auto generated by Alembic - please adjust! ###
-    op.add_column('item_tags', sa.Column('count', sa.Integer(), nullable=True))
+    op.add_column('item_tags', sa.Column('count', sa.Integer(), nullable=True, server_default=1))
```

Comment:
I'm afraid, this will still not help to update existing entries. Setting the default/server default value parameter, only applies to newly created objects - Existing objects will not be affected. 
To achieve this, you can use, `op.execute` to insert SQL statements into the migration after the new column has been created.
In this case, this should be `op.execute('update item_tags set count = 1')`
Also, the `server_default` param somehow leads to an error in the migration, so please remove it again.